pom.xml

```diff
@@ -86,10 +87,9 @@
         <!-- dependency versions -->
         <version.antlr4>4.10.1</version.antlr4>
         <version.jetty>11.0.13</version.jetty>
-        <version.logback>1.2.11</version.logback>
+        <version.logback>1.4.5</version.logback>
         <version.restassured>5.3.0</version.restassured>
         <version.jackson>2.14.1</version.jackson>
-        <version.jackson.databind>2.14.1</version.jackson.databind>
```

Comment:
Jackson should define the correct component versions, not Elide.

i.e. https://mvnrepository.com/artifact/com.fasterxml.jackson/jackson-bom/2.13.4.20221013